index.html

```diff
@@ -1837,72 +1851,70 @@
 
 <div class="issue">This section is still being refined. We expect additional principles to be added.</div>
 
-An individual may not realise when they disclose personal data that
-they are vulnerable or could become vulnerable. Some individuals
-may be more vulnerable to privacy risks or harm as a result of
-collection, misuse, loss or theft of personal data because of
-their attributes, interests, opinions or behaviour. Others may be
-vulnerable because of the situation or setting (e.g., where there
-is information asymmetry or other power imbalances), or they lack
-the capacity to fully assess the risks, or because choices are
-not presented in an easy-to-understand meaningful way (e.g., [=deceptive
-patterns=]). Yet others may be vulnerable because they have not been
-consulted about their privacy needs and expectations, or considered
-in the decisions about the design of the product of service.
-
-Sometimes communities of individuals are classed as “vulnerable”,
-typically children and the elderly, but anyone could become privacy
-vulnerable in a given context. Additional privacy protections may
-be needed for personal data of vulnerable individuals or sensitive
-information which could cause someone to become vulnerable if their
-personal data is collected, used or shared.
-
-Even in populations of individuals classed as “vulnerable” (such
-as children), each individual is unique with their own desires and
-expectations for privacy. While sometimes others can help vulnerable
-individuals assess privacy risks and make decisions about privacy
-(such as parents, guardians and peers), everyone has their own
-right to privacy.
-
 <div class="practice">
 <p>
   <span class="practicelab" id="principle-vulnerability">
-[=User agents=] and [=sites=] should allow for gracefully degraded
-user experience where some features or functionality may not be
-available because users have chosen stronger privacy protections
-(e.g., blocking tracking elements, sensor data or information about
-installed software or connected devices).
+[=User agents=] and [=sites=] should not penalize users who have chosen
+stronger privacy protections, as doing so may put vulnerable people at risk.
+If disabling some features or functionality affects user experience, it should
+degrade gracefully.
```

Comment:
```suggestion
[=User agents=] and [=sites=] should continue working if a user chooses
stronger privacy protections, to avoid putting vulnerable people at risk.
If some features or functionality cannot work for users with stronger privacy protections,
the site should degrade gracefully
```